armi/physics/neutronics/isotopicDepletion/crossSectionTable.py

```diff
@@ -154,3 +154,68 @@ def getXsecTable(
                 dataToWrite["mcnpId"] = mcnpNucName
                 output.append(tableFormat.format(**dataToWrite))
         return output
+
+
+def getCrossSectionTable(obj, nuclides=None):
```

Comment:
Since this method's main purpose was previously caching,  it might make sense to deprecate or delete. 

Right now it doesn't do anything besides provide a default kwarg.